scripts/jest/install-devtools-release-test-deps.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+/* eslint-disable no-for-of-loops/no-for-of-loops */
+
+const fs = require('fs');
+const {exec} = require('child-process-promise');
+
+// Installs an older version of React DOM into build/node_modules.
+//
+// DevTools uses React Test Renderer for its unit tests. Not React DOM. So
+// by updating React DOM to an older release, it will affect the "backend" (the
+// code that runs in the browser) but not the "frontend" (the code that runs
+// the DevTools React app itself). If we were to use React DOM in the DevTools
+// tests, we'd need to figure out a way to load separate versions for the
+// backend and the front end.
+
+const version = process.argv[2];
+
+async function main() {
+  await exec('mkdir build-tmp');
```

Comment:
We should probably have some cleanup for if this script fails midway (it does for me at the moment, and it leaves a build-tmp dir which causes it to fail next run)